### PR TITLE
Return addtlConsent, gdprApplies, tcString from tcfdata

### PR DESCRIPTION
**Pull request by ioanna0**
## What does this change?
Youtube Pfp needs some more data to pass for tcfv2 purposes 
https://developers.google.com/youtube/pfp/web/iframe_embed_config_api#ad_serving_controls
In particular they need 

- addtlConsent,
- gdprApplies, 
- tcString

This PR adds these fields so that they can be later be used in frontend inside `youtube-player.js`

### CODE REVIEW

**Review by mxdvl — COMMENTED** [review]
Looks good, but you probably want to fix the snapshots.

**Comment by ioanna0** [comment]
> Looks good, but you probably want to fix the snapshots.

@mxdvl yap i was on it, fixed now

**Comment by sndrs** [comment]
do forget to update the readme too

**Comment by mxdvl** [comment]
> @mxdvl yap i was on it, fixed now

@ioanna0, note that updating the snapshots in CI—as you did in 4f2b494—goes against making snapshots in the first place.
I’ve pinged you on a chat about this.

**Comment by sndrs** [comment]
> do forget to update the readme too

_don't_!

**Comment by ioanna0** [comment]
> @ioanna0, note that updating the snapshots in CI—as you did in [4f2b494](https://github.com/guardian/consent-management-platform/commit/4f2b4941dab829b8fa76bf7d19c9269aa4bf3e36)—goes against making snapshots in the first place.
> I’ve pinged you on a chat about this.

thanks @mxdvl ! i noticed that the snapshots are auto-generated straight after the previous commit and fixed it (will squash the commits!) but thanks for the spotting & the pinging to the chat v useful!

File: README.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ Reports the user's privacy preferences.
 type: `Object` or `undefined`
 
 Reports the user's preferences for each of the TCFv2 purposes, the last CMP
-event status and custom vendor consents.
+event status, custom vendor consents, flag if gdpr applies, the TC string and addtlConsent string.
```
Comment (mxdvl) [inline README.md:153]:
```suggestion
event status, custom vendor consents, flag if GDPR applies, the TC string and addtlConsent string.
```

**Review by mxdvl — APPROVED** [review]
👍 LGTM

Better snapshots, too! 🖼️

**Pull request merged**